crates/astria-composer/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "astria-composer"
-version = "0.2.1"
+version = "0.2.2"
```

Comment:
I think there should be an update in main cargo.toml as well?

Reply:
cargo.lock yeah - force pushed. But what about cargo.toml?

Reply:
my bad, cargo.lock what I meant